pages/freerenewal.tsx

```diff
@@ -51,7 +51,7 @@ const FreeRenewalPage: NextPage = () => {
             duration={freeRenewalDiscount.offer.duration}
             discountId={freeRenewalDiscount.offer.discountId}
             customMessage={freeRenewalDiscount.offer.customMessage}
-            priceInEth={freeRenewalDiscount.offer.price}
+            priceInEth={freeRenewalDiscount.offer.price as string}
```

Comment:
freeRenewalDiscount.offer.price isn't it already a string ?

Reply:
it's because I updated the type of price, but actually I can revert back so I will be able to remove the `as string`